plugin/trino-hive/src/test/java/io/trino/plugin/hive/parquet/ParquetTester.java

```diff
@@ -140,37 +140,47 @@
 {
     private static final int MAX_PRECISION_INT64 = toIntExact(maxPrecision(8));
 
-    private static final boolean OPTIMIZED = true;
-    private static final ConnectorSession SESSION = getHiveSession(createHiveConfig(false));
-    private static final ConnectorSession SESSION_USE_NAME = getHiveSession(createHiveConfig(true));
-    private static final List<String> TEST_COLUMN = singletonList("test");
+    protected static final boolean OPTIMIZED = true;
```

Comment:
I don't think `OPTIMIZED` is ever set to false. I think we can remove it. If we want to test non-optimized version, we should have another instance of `AbstracgtTestParquetReader`